exercises/pascals-triangle/example.rkt

```diff
@@ -0,0 +1,11 @@
+#lang lazy
```

Comment:
I think your right about why the tests are failing on CI. I think we should stick to `#lang racket`.